runtime/src/bank_forks.rs

```diff
@@ -55,6 +55,7 @@ struct SetRootTimings {
     prune_remove_ms: i64,
 }
 
+#[derive(Debug)]
```

Comment:
Think this can be removed again.

Reply:
Removing it causes the following error
```
error[E0277]: `BankForks` doesn't implement `std::fmt::Debug`
   --> runtime/src/bank.rs:823:5
    |
661 | #[derive(AbiExample, Debug)]
    |                      ----- in this derive macro expansion
...
823 |     pub loaded_programs_cache: Arc<RwLock<LoadedPrograms<BankForks>>>,
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `BankForks` cannot be formatted using `{:?}`
 ```

Reply:
Then manually: `impl Debug for LoadedPrograms {}`. Should do that anyway because it contains a lot of things that are unhelpful to display.

Reply:
Yes, agree. Would be better as a separate PR.